styles/wrap-guide.less

```diff
@@ -3,10 +3,22 @@
 atom-text-editor::shadow {
   .wrap-guide {
     height: 100%;
-    width: 1px;
+    width: 5px;
     z-index: 3;
     position: absolute;
     top: 0;
-    background-color: @syntax-wrap-guide-color;
+    background-color: transparent;
```

Comment:
Are you on OS X? You could use the zoom feature https://www.youtube.com/watch?v=jJV7CMgW5Dg to see it closer. "Smooth images" turned off and I like "Picture-in-picture".

Or take a screenshot and then zoom in:

![screen shot 2015-07-24 at 5 35 17 pm](https://cloud.githubusercontent.com/assets/378023/8871138/1fa67bf4-322e-11e5-9a3c-20e57a1cc614.png)

If you see the text edges having blue on the left and red on the right, then subpixel anti-aliasing works. So in this case it looks fine.